src/tailwind.output.css

```diff
@@ -863,10 +863,6 @@ video {
   padding-left: 0.25rem;
 }
 
-.static{
```

Comment:
Note that this change was automatically done just by running `yarn start`: I didn't actually make it.